client/src/main/java/org/apache/rocketmq/client/producer/RequestFutureTable.java

```diff
@@ -31,14 +34,22 @@
 
 public class RequestFutureTable {
```

Comment:
It seems that the name of RequestFutureTable is no longer appropriate when so many functions are put in